.travis.yml

```diff
@@ -17,6 +17,9 @@ branches:
   only:
     - master
 
+env:
```

Comment:
The env property should be set by the test-ui-run command instead, so it works from CLI everywhere.

Reply:
fixed